### PR TITLE
Issue1108 get time span tmy3

### PR DESCRIPTION
**Pull request by AnaConstantin**
for #1108 
I added a test and reference results.
Extended documentation and error output.

### CODE REVIEW

File: IBPSA/BoundaryConditions/WeatherData/ReaderTMY3.mo [hunk 1]
```diff
@@ -962,628 +962,229 @@ equation
           smooth=Smooth.Bezier,
```
Comment (mwetter) [inline IBPSA/BoundaryConditions/WeatherData/ReaderTMY3.mo]: @AnaConstantin : Can you please revert these changes in `ReaderTMY3`. I don't think they are needed, and we don't want all library developers to have them review when they merge them into their library unless there is a real change that is needed.

File: IBPSA/BoundaryConditions/WeatherData/BaseClasses/ConvertTime.mo [hunk 2]
```diff
@@ -41,7 +41,9 @@ equation
   end when;
   calTim = if canRepeatWeatherFile then modTim - (tNext - lenWea) else modTim;
   assert(canRepeatWeatherFile or (time - weaDatEndTim) < shiftSolarRad,
-    "In " + getInstanceName() + ": Insufficient weather data provided for the desired simulation period.",
+    "In " + getInstanceName() + ": Insufficient weather data provided for the desired simulation period.
+    Based on the provided weather file the following start time " + String(weaDatStaTim) +
+    " and end time " + String(weaDatEndTim) + "(last time stamp + average increment) for the weather data were determined",
```
Comment (mwetter) [inline IBPSA/BoundaryConditions/WeatherData/BaseClasses/ConvertTime.mo]: Add space between quote and bracket in `+ "(las`

File: IBPSA/BoundaryConditions/WeatherData/ReaderTMY3.mo (hunk 1 above)
Comment (AnaConstantin) [reply IBPSA/BoundaryConditions/WeatherData/ReaderTMY3.mo]: @mwetter sure. It's Dymola doing some cleaning up again. I'll double check next time.

File: IBPSA/BoundaryConditions/WeatherData/BaseClasses/ConvertTime.mo (hunk 2 above)
Comment (AnaConstantin) [reply IBPSA/BoundaryConditions/WeatherData/BaseClasses/ConvertTime.mo]: Thank you. Done!

File: IBPSA/BoundaryConditions/WeatherData/ReaderTMY3.mo (hunk 1 above)
Comment (AnaConstantin) [reply IBPSA/BoundaryConditions/WeatherData/ReaderTMY3.mo]: Thank you. Done! It should now show only the additional documentation I wrote.
